ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -22,24 +22,51 @@
           aria-label={{t 'actions.search'}}
           {{on 'input' this.handleSearchInput}}
         />
-        <S.Dropdown as |DD|>
+        <S.Dropdown @onClose={{this.close}} as |DD|>
           <DD.ToggleButton
             @color='secondary'
             @text={{t 'resources.scope.title'}}
+            @count={{this.scopes.length}}
           />
           <DD.Header @hasDivider={{true}}>
             <Hds::Form::TextInput::Base
               @type='search'
+              @value={{this.scopeSearchTerm}}
               placeholder={{t 'actions.narrow-results'}}
               aria-label={{t 'actions.narrow-results'}}
+              {{on 'search' this.filterItems}}
```

Comment:
Shouldn't this be on `input`?

Reply:
This is the search input inside the dropdown.

Reply:
Right, why isn't it on input as well? And I don't think [search](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/search_event) can be used in production.